### PR TITLE
tech: fixed deprecated issues

### PR DESCRIPTION
**Pull request by RalucaOT**
Fixed deprecated issues after verified compatibility using intelliJ plugin verifier.

### CODE REVIEW

File: src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/actions/PhaseComboBoxAction.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void update(final AnActionEvent event) {
 
     @NotNull
     @Override
-    protected DefaultActionGroup createPopupActionGroup(JComponent button) {
+    protected DefaultActionGroup createPopupActionGroup(JComponent button, DataContext dataContext) {
```
Comment (vladbenchea) [inline src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/actions/PhaseComboBoxAction.java:63]: why is this required? param isn't used anywhere as I see in here

Comment (RalucaOT) [reply src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/actions/PhaseComboBoxAction.java:63]: https://github.com/JetBrains/intellij-community/blob/master/platform/platform-api/src/com/intellij/openapi/actionSystem/ex/ComboBoxAction.java

image

**Review by vladbenchea — APPROVED** [review]

**Review by pikachugb — APPROVED** [review]
